ledger/participant-state/kvutils/tools/src/main/scala/ledger/participant/state/kvutils/tools/integritycheck/Config.scala

```diff
@@ -17,6 +17,8 @@ case class Config(
     jdbcUrl: Option[String] = None,
     expectedUpdatesPath: Option[Path] = None,
     actualUpdatesPath: Option[Path] = None,
+    expectedUpdatesNormalizers: Seq[UpdateNormalizer] = Seq.empty,
```

Comment:
Minor: you might want to keep the first word plural, i.e., `expectedUpdateNormalizers`. Also, an `Iterable[UpdateNormalizer]` should suffice here.

Reply:
Usually yes but I wanted to make it consistent with the previous params. I'll open another PR with this change.